data/inspect.csv

```diff
@@ -1313,12 +1313,12 @@ veterans.gov,http://www.veterans.gov,True,True,http://www.dol.gov/vets/,False,Fa
 vetsuccess.gov,http://vetsuccess.gov,False,False,,False,False,False,False,False,False,False,,,False,False,False,True,True
 visitthecapital.gov,http://www.visitthecapital.gov,True,False,,False,False,False,True,False,True,False,,,False,False,False,False,False
 visitthecapitol.gov,https://www.visitthecapitol.gov,True,False,,True,True,False,True,False,False,False,,,False,False,False,False,False
-vista.gov,http://www.vista.gov,True,True,http://www.nationalservice.gov/programs/americorps,False,False,False,False,False,False,False,,,False,False,False,False,False
+vista.gov,http://www.vista.gov,True,True,http://www.nationalservice.gov/programs/americorps,False,False,False,False,True,True,False,,,False,False,False,False,False
 vistacampus.gov,http://www.vistacampus.gov,True,False,,False,False,False,False,False,True,False,,,False,False,False,False,False
 vitm.gov,http://vitm.gov,False,False,,False,False,False,False,False,False,False,,,False,False,False,True,True
 voa.gov,http://www.voa.gov,True,True,http://www.voanews.com/,False,False,False,False,False,True,False,,,False,False,False,False,False
 volcano.gov,http://volcano.gov,False,False,,False,False,False,False,False,False,False,,,False,False,False,True,True
-volunteer.gov,https://www.volunteer.gov,True,False,,True,True,False,True,False,False,False,,,False,False,False,False,False
+volunteer.gov,http://www.volunteer.gov,True,False,,False,False,False,False,False,False,False,,,False,False,False,False,False

```

Comment:
`volunteer.gov` appears to force HTTPS now. So, a note to ignore this if it shows up as forcing again next week.